helm/reana/values.yaml

```
@@ -53,6 +53,10 @@ secrets:
   cern:
     sso: {}
   reana: {}
+  login: {}
+
+# Enable and configure SSO authentication via a third-party Keycloak identity provider
+login: {}
```

```suggestion
login: []
```
The default value should be `[]` instead of `{}`, what do you think?